pkg/util/config/feature_gate.go

```diff
@@ -34,20 +34,29 @@ const (
 	// a featureSpec entry to knownFeatures.
 
 	// allAlphaGate is a global toggle for alpha features. Per-feature key
-	// values override the default set by allAlphaGate, if they come later in the
-	// specification of gates. Examples:
+	// values override the default set by allAlphaGate. Examples:
 	//   AllAlpha=false,NewFeature=true  will result in newFeature=true
-	//   AllAlpha=true,NewFeature=false  will result in newFeature=false
-	allAlphaGate         = "AllAlpha"
-	dynamicKubeletConfig = "DynamicKubeletConfig"
+	//   NewFeature=false,AllAlpha=true will result in newFeature=false
+	allAlphaGate = "AllAlpha"
+
+	// DynamicKubeletConfig is a toggle for kubelet loading its
+	// config via a config map from the apiserver.
+	dynamicKubeletConfigGate = "DynamicKubeletConfig"
```

Comment:
A const with suffix `Gate` denotes a key in the `--feature-gates` map. I actually like this pattern, so keeping it unless more people complain.